perf_test/blas/blas/KokkosBlas_trtri_perf_test.hpp

```diff
@@ -78,6 +78,21 @@ void (*do_trtri_invoke[LOOP_N][TEST_N])(options_t) = {
 /*************************** Test types and defaults **************************/
 #define DEFAULT_TRTRI_ARGS "UU"
 
+  /**
+   * The KokkosBatched::SerialTrtri implementation performs trmm and scal on subblocks
+   * of the A matrix. a_m subblocks are selected.
+   */
+static inline int trtri_flop_count(int a_m, int a_n) {
+  int flop_count = 0;
+
+  for (int i = 0; i < a_m; i++) {
+    flop_count++;                // 1 / A[i,j]
+    flop_count += (i * (i + 1)); // TRMM FLOPS
+    flop_count += i;             // SCAL FLOPS
+  }
+  return flop_count;
+}
+
```

Comment:
Can we replace with n^3/3 + 2*n/3?

Reply:
By the way, this is the "paper-and-pencil" flop count for trtri (e.g., LAPACK working note 41)..